src/utils.ts

```diff
@@ -25,6 +55,21 @@ export function parseUrl(url: string): Url {
   return urlObject;
 }
 
+/**
+ * Ensure path separators of the posix forward-slash form when
+ * running on environment (Windows) where path separator is a
+ * backslash.  Backslash *is* a valid filename character in a
+ * posix environment, so this function is a no-op there.
+ *
+ * @param pathname - file path to transform.
+ */
+export function posixify(pathname: string): string {
```

Comment:
Is it a problem that a round-trip between `posixify` and `normalizePathSeparators` is potentially lossy? Should we throw in cases where the input contains a `/` on windows?